torch/csrc/jit/import_source.cpp

```diff
@@ -141,7 +146,7 @@ struct SourceResolver : public Resolver {
   }
 
   TypePtr resolveType(const std::string& name, const SourceRange& loc) const override {
-    return lib_cu_.get_class(c10::QualifiedName(name));
+    return lib_cu_.get_serializable_type(c10::QualifiedName(name));
```

Comment:
Yeah, we should coordinate how this will work across interfaces and named tuples, both require extending this API.